tests/framework/extensions/psact/createdeployment.go

```diff
@@ -68,12 +69,16 @@ func CreateNginxDeployment(client *rancher.Client, clusterID string, psact strin
 			return false, err
 		}
 
-		if *deployment.Spec.Replicas == deployment.Status.AvailableReplicas && (psact == string(provisioninginput.RancherPrivileged) || psact == string(provisioninginput.RancherBaseline)) {
+		if psact == string(provisioninginput.RancherRestricted) {
+			for _, condition := range deployment.Status.Conditions {
+				if strings.Contains(condition.Message, `forbidden: violates PodSecurity "restricted:latest"`) {
```

Comment:
nit; should we have the message be in a `const` variable or is it fine as is?

Reply:
done